src/ClassMethodsHydrator.php

```diff
@@ -194,6 +179,27 @@ public function extract(object $object) : array
         return $values;
     }
 
+    private function initCompositeFilter($object) : Filter\FilterComposite
```

Comment:
we can add typehint `object` on `$object` parameter

Reply:
I just learned that this was added to 7.2. I applied it.